Source/Krypton Components/TestForm/TestForm.csproj

```diff
@@ -45,6 +45,7 @@
             <Version>4.0.0.0</Version>
             <!-- Designers for the `TFMs != net4` are "Pulled in" via Visual studios ".nuget\packages\microsoft.windowsdesktop.app.ref" -->
         </Reference>
+        <Reference Include="System.Windows.Forms" />
```

Comment:
Is this because the `usingWinforms` was removed from the top part ?

Reply:
Yes, though more investigation is needed